### ISSUE TITLE
Contributions are not working on dashboard

### ISSUE THREAD

**Issue by matentzn** [cthoyt/obo-community-health#19]
The contribution part of the dashboard is right now not working? 

![image](https://github.com/cthoyt/obo-community-health/assets/7070631/6fe70bb1-ae2d-4a08-b575-573d5b5ff37c)

**Comment by cthoyt** [comment]
The GitHub API is unstable and sometimes doesn't return these data

**Comment by matentzn** [comment]
I assume these numbers are rebuild during the weekly refresh, maybe let it fail if the numbers are not generated?

**Comment by cmungall** [comment]
still an issue?

**Comment by cthoyt** [comment]
Yes, I just tried this again and GitHub's API for activities on a repository is still weirdly unstable
